pkg/kubectl/cmd/log.go

```diff
@@ -93,6 +93,7 @@ func NewCmdLog(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 		Aliases: []string{"log"},
 	}
 	cmd.Flags().BoolVarP(&o.Follow, "follow", "f", o.Follow, "Specify if the logs should be streamed.")
+	cmd.Flags().MarkShorthandDeprecated("follow", "please use --follow instead")
```

Comment:
@bgrant0607 who made this rule that shorthand flags can't be used in appropriate contexts?  I think --follow here is far less friendly than -f, and -f is not at all confusing